ext/RMagick/rmenum.c

```diff
@@ -402,39 +402,47 @@ ColorspaceType_new(ColorspaceType cs)
 
     switch(cs)
     {
-        default:
-        ENUM_SET_NAME(UndefinedColorspace)
-        ENUM_SET_NAME(RGBColorspace)
+        ENUM_SET_NAME(CMYColorspace)
+        ENUM_SET_NAME(CMYKColorspace)
         ENUM_SET_NAME(GRAYColorspace)
-        ENUM_SET_NAME(TransparentColorspace)
+#if defined(IMAGEMAGICK_GREATER_THAN_EQUAL_6_8_9)
```

Comment:
I don't prefer to split `IMAGEMAGICK_GREATER_THAN_EQUAL_6_8_9` block. It is ugly for me.

Reply:
So, if you want to sort the order, I think you should do outside the `IMAGEMAGICK_GREATER_THAN_EQUAL_6_8_9` block.

Reply:
I decided to use this approach because there will be a lot of `#if defined(IMAGEMAGICK_7)` or `#if !defined(IMAGEMAGICK_7)` for the ImageMagick 7 changes. And that will make it more consistent. If you would prefer to have a separate block for `IMAGEMAGICK_7` also then I can change this back to a block again.

Reply:
I would prefer this solution but I will let you decide @Watson1978.